src/common/threadsafe_queue.h

```diff
@@ -101,7 +98,7 @@ class SPSCQueue {
     // and a pointer to the next ElementPtr
     class ElementPtr {
     public:
-        ElementPtr() : next(nullptr) {}
+        ElementPtr() {}
```

Comment:
```suggestion
        ElementPtr() = default;
```
or probably just remove this?